apps/staking/src/hooks/use-api.tsx

```diff
@@ -95,7 +96,7 @@ const State = {
       dashboardDataCacheKey,
 
       loadData: () =>
-        api.loadData(client, pythnetClient, hermesClient, account),
+        api.loadData(client, pythnetClient, hermesClient, account, simulationPayer ? new PublicKey(simulationPayer) : undefined),
```

Comment:
I'm not sure where to put the string to public key conversion.
Ideally the deployment should fail if SIMULATION_PAYER is an invalid base58 pubkey